libraries/rush-lib/src/logic/EventHooksManager.ts

```diff
@@ -53,7 +53,7 @@ export class EventHooksManager {
           console.error(
             '\n' +
               colors.yellow(
-                `Event hook "${script}" failed. Run "rush" with --debug` +
+                `Event hook "${script}" failed: ${error}\nRun "rush" with --debug` +
```

Comment:
This was especially useful for testing, especially since the -debug flag didn't really help in my case (script name had a typo).    Because it was so useful, I think it should stay in.